src/rust/engine/process_execution/src/remote_cache.rs

```diff
@@ -144,6 +145,7 @@ impl CommandRunner {
     let mut file_digests = Vec::new();
     sub_trie.walk(&mut |_, entry| match entry {
       directory::Entry::File(f) => file_digests.push(f.digest()),
+      directory::Entry::Symlink(_) => todo!(),
```

Comment:
Do nothing, this loop gathers file digests that the caller will need to do something with. Symlinks don't have a digest.